lib/client/client.js

```diff
@@ -1005,6 +1013,8 @@ Client.prototype.connect = function connect () {
     // Communicate the last-encountered error
     if (err instanceof ConnectionError) {
       self.emit('connectTimeout', err)
+    } else if (err.code === 'ECONNREFUSED') {
+      self.emit('connectRefused', err)
```

Comment:
is this supposed to be `'connectError'` like above?

Reply:
All the tests pass when I comment out this line. Could you write a test that makes sure this emits the correct event?